### ISSUE TITLE
App bar focus on panel open

### ISSUE THREAD

**Issue by jolevesq** [Canadian-Geospatial-Platform/GeoView#44]
When a panel opens from app bar (and any other panel), focus should be set on the open panel close button. This piece of code should be added at the custom panel component level (#6) so all implementation of panel will have it set.

At the same time, on creation this panel should receive a reference to it's caller so when the panel close, the focus is set on the caller.

**Comment by jolevesq** [comment]
Esc key should close open panel
